ignite/handlers/logging.py

```diff
@@ -17,10 +16,10 @@ def log_training_simple_moving_average(trainer, window_size, history_transform=l
 def log_validation_simple_moving_average(trainer, window_size, history_transform=lambda x: x,
                                          should_log=lambda trainer: True, metric_name="", logger=print):
     if should_log(trainer):
-        total_iterations = len(trainer.validation_data)
-        current_iteration = trainer.current_iteration % total_iterations
+        total_iterations = trainer.total_validation_iterations
+        current_iteration = trainer.current_iteration % (total_iterations + 1)
```

Comment:
strange, this should be `trainer.current_validation_iteration` not `trainer.current_iteration`

Reply:
Same for other validation logging functions